### PR TITLE
Add a new documentation page specific for expression variables

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

File: documentation/reference/expression_variables.en.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  - `@position_source_name` > Reports the sensor name.
+  
+- When the crosshair is manually moved:
+  - `@gnss_horizontal_accuracy` > Reports the sensor's horizontal accuracy (in meters).
+  - `@position_horizontal_accuracy` > The value is `NULL`.
+  - `@position_source_name` > The value is `manual`.
+
+## QFieldCloud
+
+For QFieldCloud users, two expression variables can be used in forms or default values:
```
Comment (nirvn) [inline documentation/reference/expression_variables.en.md:60]: Nice, good move overall.

File: documentation/reference/expression_variables.en.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- `@position_fix_status_description` - The GPS fix status ("NoData", "NoFix", "Fix2D", or "Fix3D"), as reported by the sensor. Available only when the crosshair is snapped to the sensor.
+- `@position_fix_mode` - The fix mode ("M" = Manual, "A" = Automatic), as reported by the sensor. Available only when the crosshair is snapped to the sensor.
+
+!!! info
+    - I: Internal position source, E: External (NMEA) position source.
+    - Variables containing `satellites` are not available on iOS.
+
+### Examples of Usage
+
+- When the crosshair is snapped to the sensor:
```
Comment (nirvn) [inline documentation/reference/expression_variables.en.md:48]: When the crosshair is _not_ snapped

File: documentation/reference/expression_variables.en.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  - `@gnss_horizontal_accuracy` > Reports the horizontal accuracy (in meters) from the sensor.
+  - `@position_horizontal_accuracy` > Reports the same value if the crosshair is snapped to the sensor.
+  - `@position_source_name` > Reports the sensor name.
+  
+- When the crosshair is manually moved:
```
Comment (nirvn) [inline documentation/reference/expression_variables.en.md:53]: Is this a repetition of the above?

**Comment by m-kuhn** [comment]
@nirvn better like this?

**Review by suricactus — APPROVED** [review]
Good move!

**Pull request merged**